infra/stacks/ssm.ts

```
@@ -0,0 +1,22 @@
+import * as secrets from "./secrets";
```

ESLint

[error] Resolve error: EACCES: permission denied, open '/nEcTisZEIa'
    at Object.writeFileSync (node:fs:2409:20)
    at l (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:13685)
    at createFilesMatcher (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:14437)
    at Object.resolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import@2.32.0_eslint@9.35.0_jiti@2.5.1_/node_modules/eslint-import-resolver-typescript/lib/index.cjs:298:107)
    at withResolver (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.43.0_eslint@9.35.0_jiti@2.5.1__t_2d081e10e311ce89b8ed867371a1e5c0/node_modules/eslint-module-utils/resolve.js:180:23)
    at fullResolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.43.0_eslint@9.35.0_jiti@2.5.1__t_2d081e10e311ce89b8ed867371a1e5c0/node_modules/eslint-module-utils/resolve.js:201:22)
    at relative (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.43.0_eslint@9.35.0_jiti@2.5.1__t_2d081e10e311ce89b8ed867371a1e5c0/node_modules/eslint-module-utils/resolve.js:217:10)
    at resolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.43.0_eslint@9.35.0_jiti@2.5.1__t_2d081e10e311ce89b8ed867371a1e5c0/node_modules/eslint-module-utils/resolve.js:233:12)
    at ExportMapBuilder.get (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.43.0_eslint@9.35.0_jiti@2.5.1___dc19963699076fe853eebdef9d9a2d89/node_modules/eslint-plugin-import/lib/exportMap/builder.js:37:45)
    at processBodyStatement (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.43.0_eslint@9.35.0_jiti@2.5.1___dc19963699076fe853eebdef9d9a2d89/node_modules/eslint-plugin-import/lib/rules/namespace.js:12:38)

(import/namespace)